docs/reference/api.rst

```diff
@@ -111,6 +111,9 @@ Plugins
 .. automodapi:: jdaviz.configs.imviz.plugins.aper_phot_simple.aper_phot_simple
    :no-inheritance-diagram:
 
+.. automodapi:: jdaviz.configs.imviz.plugins.catalogs.catalogs
```

Comment:
That's weird. I remember suggesting this to Sabrina's PR but maybe it was lost in rebase. Thanks for adding this.